framework/physics/physics_namespace.h

```diff
@@ -13,7 +13,7 @@ enum class OperationType
 {
   SINGLE_VALUE = 0,
   FROM_ARRAY = 1,
-  SIMPLEXS0 = 20,
+  PURE_ABSORBER = 20,
   SIMPLEXS1 = 21,
```

Comment:
The name here is a bit tricky because the XS are not in a standard form that has a common name. It may be good to add a `DOWN_SCATTERER` option and call this `FULL_SCATTERER`, `UPDOWN_SCATTERER`, or `MIXED_SCATTERER` or something like that.